--- origsrc/libgda-5.1.2/m4/bdb.m4	2013-12-06 02:20:50.234908000 -0600
+++ src/libgda-5.1.2/m4/bdb.m4	2013-12-06 00:48:05.635631200 -0600
@@ -114,14 +114,9 @@ m4_define([_BDB_CHECK_INTERNAL],
 	do
 	    bdbdir=""
 	    AC_MSG_CHECKING([for Berkeley DB files in $d])
-	    for version in "" 5.1 5.0 4.9 4.8 4.7
+	    for version in 5.3 5.2 5.1 5.0 4.9 4.8 4.7
 	    do
-	        if test $platform_win32 = yes
-		then
-		    sversion=`echo $version | sed -e 's,\.,,g'`
-		else
-		    sversion=`echo $version | sed -e 's,\..*,,g'`
-		fi
+		sversion=`echo $version | sed -e 's,\..*,,g'`
 		if test -z $version
 		then
 		    db_libfilename="libdb$bdb_libext"
@@ -129,18 +124,10 @@ m4_define([_BDB_CHECK_INTERNAL],
 		    db_libfile="$d/$bdb_loclibdir/libdb$bdb_libext"
 		    try_headers="db.h"
     		else
-		    if test $platform_win32 = yes
-		    then
-		        db_libfilename="libdb$sversion$bdb_libext"
-        	        db_lib="-ldb$sversion"
-		        db_libfile="$d/$bdb_loclibdir/libdb$sversion$bdb_libext"
-        	        try_headers="db.h db$version/db.h db$sversion/db.h"
-		    else
 		        db_libfilename="libdb-$version$bdb_libext"
         	        db_lib="-ldb-$version"
 		        db_libfile="$d/$bdb_loclibdir/libdb-$version$bdb_libext"
         	        try_headers="db.h db$version/db.h db$sversion/db.h"
-    		    fi
 		fi
 
 		for db_hdr in $try_headers
