--- origsrc/libgda-5.2.2/m4/bdb.m4	2013-11-16 08:27:35.000000000 -0600
+++ src/libgda-5.2.2/m4/bdb.m4	2015-01-08 11:41:04.333028600 -0600
@@ -114,7 +114,7 @@ m4_define([_BDB_CHECK_INTERNAL],
 	do
 	    bdbdir=""
 	    AC_MSG_CHECKING([for Berkeley DB files in $d])
-	    for version in "" 6.0 5.3 5.1 5.0 4.9 4.8 4.7
+	    for version in 6.0 5.3 5.2 5.1 5.0 4.9 4.8 4.7
 	    do
 	        if test $platform_win32 = yes
 		then
@@ -129,7 +129,13 @@ m4_define([_BDB_CHECK_INTERNAL],
 		    db_libfile="$d/$bdb_loclibdir/libdb$bdb_libext"
 		    try_headers="db.h"
     		else
-		    if test $platform_win32 = yes
+		    if test $platform_cygwin = yes
+		    then
+		        db_libfilename="cygdb-$version.dll"
+        	        db_lib="-ldb-$version"
+		        db_libfile="$d/lib/libdb-$version.dll.a"
+        	        try_headers="db$version/db.h db$sversion/db.h"
+		    elif test $platform_win32 = yes
 		    then
 		        db_libfilename="libdb$sversion$bdb_libext"
         	        db_lib="-ldb$sversion"
